### ISSUE TITLE
Remove tool/coords

### ISSUE THREAD

**Issue by adamdoda** [UOAF/project-tauntaun#134]
The semi major axes values can be found in DCS somewhere so in theory after using that this script can be removed.

**Comment by bobmoretti** [comment]
Does this mean we can also remove `requirements-tools.txt`?

**Comment by adamdoda** [comment]
Yes but let's put this on hold maybe we need the script to calculate the map offsets and zone id. I don't remember what it did exactly.

**Comment by adamdoda** [comment]
I think liberation needs to do the same so probably we could steal how they do it.
